front/app/services/projects.ts

```diff
@@ -359,6 +359,15 @@ export async function deleteProject(projectId: string) {
   return response;
 }
 
+export async function copyProject(projectId: string) {
+  const response = await streams.add(`${apiEndpoint}/${projectId}/copy`, {});
+
+  await streams.fetchAllWith({
+    apiEndpoint: [`${API_PATH}/projects`, `${API_PATH}/admin_publications`],
```

Comment:
We may not need to refetch the projects here. I think only adminPublications are used on the overview page. (Also, I'd only add these if there is a problem, not by default.)

Reply:
Indeed, only `admin_publications` are needed for the page where it's going to be used. But what if an admin copies a project and then goes to another page where the `projects` endpoint is used? `projects` page will be stale in this case.

Reply:
You can test it here https://github.com/CitizenLabDotCo/citizenlab/pull/3878